docs/issues/disposal-rule-apply-vs-preview-matching.md

```
@@ -0,0 +1,34 @@
+# Gallringsregler: "Tillämpa regler" matchade annorlunda än förhandsgranskningen
+
+## Problem
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)